ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/Command.java

```diff
@@ -26,17 +26,20 @@
 public class Command {
   private final String statement;
   private final boolean useOffsetAsQueryID;
+  private final boolean neverSkipCommand;
   private final Map<String, Object> overwriteProperties;
   private final Map<String, String> originalProperties;
   private final boolean preVersion5;
 
   @JsonCreator
   public Command(@JsonProperty("statement") final String statement,
                  @JsonProperty("useOffsetAsQueryID") final Boolean useOffsetAsQueryID,
+                 @JsonProperty("neverSkipCommand") final Boolean neverSkipCommand,
```

Comment:
let's introduce this as a breaking change since we're not version 1.0 yet and we're adding a ton of breaking changes

Reply:
+1

Reply:
Now that this is a breaking change, we don't need this flag anymore

Reply:
Oh I see, I misunderstood what you two meant by the breaking change, I got rid of it.